### PR TITLE
[C# & TypeScript] Update BF SDK versions to 4.11.0

### LINKED ISSUE #3706: [C# & TypeScript] Update BF SDK versions to 4.11.0
## User Story

As BF SDK 4.11 versions were published for [JavaScript](https://github.com/microsoft/botbuilder-js/releases/tag/4.11.0) and [C#](https://github.com/microsoft/botbuilder-dotnet/releases/tag/4.11.0)
 I want to have the C# and TypeScript solutions updated
 so that the C# and TypeScript solutions uses the latest SDK version published

## Acceptance Criteria

*List your requirements.*

- [x] Update the `.csproj` files of the C# solutions to use BF SDK version 4.11.0
- [x] Update the `package.json` and `package-lock.json` files of the TypeScript solutions to use BF SDK version 4.11.0
- [x] Replicate changes to the templates
- [x] Validate the changes

## Resources

*Provide any screenshots or conversation mockups to support your request.*
 *Learn how to mock up bot conversations with [.chat files](http://aka.ms/bfchatfile).*

- [C# Release](https://github.com/microsoft/botbuilder-dotnet/releases/tag/4.11.0)
- [JS Release](https://github.com/microsoft/botbuilder-js/releases/tag/4.11.0)

## Notes

### PR DESCRIPTION
**Pull request by matiasroldan6**
Fixes #3706

### Purpose

*What is the context of this pull request? Why is it being done?*
 The last version published of BF SDK is 4.11.0

### Changes

*Are there any changes that need to be called out as significant  or particularly difficult to grasp? (Include illustrative screenshots  for context if applicable.)*

- Update the `package.json` and `package-lock.json` files to use 4.11.0 for the BF libraries for TypeScript solutions
- Update the `.csproj` files to use 4.11.0 for the BF libraries for C# solutions

### Tests

*Is this covered by existing tests or new ones? If no, why not?*
 Yes, it was tested manually and validated the build and tests execution of the packages.

### Feature Plan

*Are there any remaining steps or dependencies before this issue  can be fully resolved? If so, describe and link to any relevant pull  requests or issues.*
 \-

#### General

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated the appropriate tests
- [ ] I have updated related documentation

### CODE REVIEW

File: samples/csharp/assistants/enterprise-assistant/VirtualAssistantSample/VirtualAssistantSample.csproj [hunk 1]
```diff
@@ -14,13 +14,14 @@
...
-    <PackageReference Include="Microsoft.Bot.Builder.LanguageGeneration" Version="4.9.1" />
+    <PackageReference Include="Microsoft.Bot.Builder.AI.Luis" Version="4.11.0" />
+    <PackageReference Include="Microsoft.Bot.Builder.AI.QnA" Version="4.11.0" />
+    <PackageReference Include="Microsoft.Bot.Builder.ApplicationInsights" Version="4.11.0" />
+    <PackageReference Include="Microsoft.Bot.Builder.Azure" Version="4.11.0" />
+    <PackageReference Include="Microsoft.Bot.Builder.Azure.Blobs" Version="4.11.0" />
+    <PackageReference Include="Microsoft.Bot.Builder.Integration.ApplicationInsights.Core" Version="4.11.0" />
+    <PackageReference Include="Microsoft.Bot.Builder.Integration.AspNet.Core" Version="4.11.0" />
+    <PackageReference Include="Microsoft.Bot.Builder.LanguageGeneration" Version="4.11.0" />
     <PackageReference Include="Microsoft.Bot.Solutions" Version="1.0.0" />
```
Comment (peterinnesmsft) [inline samples/csharp/assistants/enterprise-assistant/VirtualAssistantSample/VirtualAssistantSample.csproj]: Can we update this to 1.0.1 while we're at it?

**Review by peterinnesmsft — APPROVED** [review]

File: samples/csharp/assistants/enterprise-assistant/VirtualAssistantSample/VirtualAssistantSample.csproj (hunk 1 above)
Comment (Batta32) [reply samples/csharp/assistants/enterprise-assistant/VirtualAssistantSample/VirtualAssistantSample.csproj]: Sure @peterinnesmsft ! We already updated to bot-solutions version 1.0.1, also double checked the others `.csproj` files 😊.

File: sdk/typescript/libraries/bot-solutions/package.json [hunk 2]
```diff
@@ -26,13 +26,13 @@
...
-        "botbuilder-ai": "^4.9.2",
-        "botbuilder-azure": "^4.9.2",
-        "botbuilder-dialogs": "^4.9.2",
-        "botframework-config": "^4.9.2",
-        "botframework-connector": "^4.9.2",
-        "botbuilder-lg": "^4.9.2",
+        "botbuilder": "^4.11.0",
+        "botbuilder-ai": "^4.11.0",
+        "botbuilder-azure": "^4.11.0",
+        "botbuilder-dialogs": "^4.11.0",
```
Comment (dawwa) [inline sdk/typescript/libraries/bot-solutions/package.json:32]: Is this intended not to be 4.11.1 as in the csproj files?

Comment (Batta32) [reply sdk/typescript/libraries/bot-solutions/package.json:32]: @dawwa - 4.11.1 was published in C# (see [NuGet > BotBuilder](https://www.nuget.org/packages/Microsoft.Bot.Builder/4.11.1)), however, in JS it was not published (see [npmjs > botbuilder](https://www.npmjs.com/package/botbuilder)).

As soon as we detect that a new package was published, we will update these references. However, the `^` character will install the last package version published automatically.

**Review by darrenj — APPROVED** [review]

**Comment by lauren-mills** [comment]
Closing - outdated

**Pull request closed without merging**